compiler/quilt/tools/command.py

```diff
@@ -1370,6 +1370,9 @@ def delete_user(username, force=False, team=None):
     url = get_registry_url(team)
     resp = session.post('%s/api/users/delete' % url, data=json.dumps({'username':username}))
 
+def delete_user_cli(username, force=False):
```

Comment:
Why is this only needed for delete - and not create or disable?

Also, by requiring the team, I meant, it needs to be specified, not `None` - otherwise, that's kinda the same thing.

Reply:
because delete also has force as an optional parameter